openwisp_radius/admin.py

```diff
@@ -555,6 +561,18 @@ class AlwaysHasChangedForm(AlwaysHasChangedMixin, forms.ModelForm):
         choices=OPTIONAL_FIELD_CHOICES,
         fallback=OPTIONAL_SETTINGS.get('birth_date', 'disabled'),
     )
+    registration_api_enabled = FallbackNullChoiceField(
```

Comment:
Let's just call this `registration_enabled`.
Ensure this field is shown before `first_name`.